litestar/handlers/http_handlers/decorators.py

```diff
@@ -82,7 +82,7 @@ def __init__(
         content_media_type: str | None = None,
         deprecated: bool = False,
         description: str | None = None,
-        include_in_schema: bool = True,
+        include_in_schema: bool | EmptyType | None = Empty,
```

Comment:
Why none? 

I get why the empty type but not what's none here

Reply:
I followed the type hints used in `return_dto: type[AbstractDTO] | None | EmptyType = Empty`, I did find the usage of None for `include_in_schema` rather pointless, so I will make it `bool | Empty`

Reply:
Yes, the typing there is meant to allow explicitly overriding the DTO on lower layers and setting it the none. It's not the best API tbh.